### PR TITLE
navigate to welcome page after finishing feature tour

### PR DESCRIPTION
**Pull request by Eskibear**
Part of #540 , in case users don't click the button, we navigate to welcome page after 5 seconds.

### CODE REVIEW

**Review by jdneo — APPROVED** [review]

File: src/welcome/assets/components/TourPage.tsx [hunk 1]
```diff
@@ -17,35 +17,48 @@ export default class TourPage extends Component<{
...
+            ...this.getContentPages(),
+            this.getEndingPage()
+        ];
+
+        this.timer = undefined;
     }
 
+    private steps: JSX.Element[];
+    // auto navigate to welcome page in last step
+    private timer: number | undefined;
```
Comment (jdneo) [inline src/welcome/assets/components/TourPage.tsx:32]: Shouldn't it declared as `NodeJS.Timer`?

Comment (Eskibear) [reply src/welcome/assets/components/TourPage.tsx:32]: `NodeJS.Timer` failed webpack compilation.  In Javascript, `settimeout` returns id of the timer, thus it's a `number`. In NodeJS, it's a `Timer`.  Here this is used in a webview, so `number` type in plain JS should be used.

Comment (jdneo) [reply src/welcome/assets/components/TourPage.tsx:32]: I see. That makes sense.

**Pull request merged**